angr/analyses/stack_pointer_tracker.py

```diff
@@ -273,6 +273,8 @@ def _offset_for(self, addr, pre_or_post, reg):
             regval = dict(self._state_for(addr, pre_or_post).regs)[reg]
         except KeyError:
             return TOP
+        except AttributeError:
+            return TOP
```

Comment:
Just curious: When does `AttributeError` occur? Seems to be a bug that we should fix instead of catching it and silently returning a `TOP`.

Reply:
@ltfish remember the other day when I said "you're not going to like this, and should look into why there are Nones flying around"? This is what I was referring to. A None makes it in here, and I couldn't figure out why. Some super deep voodoo, but it only happened after your FunctionManager fix got pulled in

Reply:
lol
OK time to debug it.

Reply:
I took this commit out and I will help you fix the cases where that exception is triggered.